kubernetesevents/delta_fifo.go

```diff
@@ -105,9 +107,14 @@ func (d *DeltaFIFO) Process() {
 
 	go func(done chan error) {
 		wait := 1
-		dialer := &websocket.Dialer{}
+		dialer := &websocket.Dialer{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
```

Comment:
We do, but I haven't found a way to specify the cert using the real Kubernetes client so I did it this way here to just be consistent.

The Kubernetes client only allows the cert on the `user` field and so it assumes the cert is also used for authentication. We use a bearer token so I'm not sure how to do both of these at the same time.